_sass/_copyable.scss

```diff
@@ -0,0 +1,32 @@
+.copyable {
```

Comment:
On chrome: 
![image](https://cloud.githubusercontent.com/assets/123822/24296878/22db2ae2-10a1-11e7-8567-13dab3b2c4c5.png)

Can we remove the outline 😱